src/constants/assets.ts

```diff
@@ -111,9 +111,9 @@ export const ASSETS_DICT: { [key in AssetSymbol]: Asset } = {
   },
   aSEED: {
     symbol: 'aSEED',
-    displaySymbol: 'aUSD',
-    name: 'Acalla Dollar',
-    icon: SymbolAusd,
+    displaySymbol: 'aSEED',
+    name: 'Acala Dollar',
```

Comment:
Please change to the new name `aUSD SEED`,
https://wiki.acala.network/acala-exodus-upgrade/ausd-seed-aseed#ausd-conversion-date